packages/admin/src/GlobalSearch/Actions/Action.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Filament\GlobalSearch\Actions;
+
+use Filament\GlobalSearch\Actions\Concerns\CanEmitEvent;
+use Filament\Support\Actions\BaseAction;
+use Filament\Support\Actions\Concerns\CanBeOutlined;
+use Filament\Support\Actions\Concerns\CanOpenUrl;
+use Illuminate\Contracts\Support\Arrayable;
+use Illuminate\Support\Str;
+
+class Action extends BaseAction implements Arrayable
+{
+    use CanBeOutlined;
+    use CanEmitEvent;
+    use CanOpenUrl;
+
+    protected string $view = 'filament::global-search.actions.link-action';
+
+    protected string $viewIdentifier = 'action';
+
+    public function toArray(): array
```

Comment:
I don't think the actions need to be fully serializable like in notifications, right? Could just be similar to actions in the tables and forms package.

Reply:
Really don't need it